docs/EmbeddedAppAndroid.md

```diff
@@ -166,7 +129,3 @@ To run your app, you need to first start the development server. To do this, sim
 Now build and run your Android app as normal (e.g. `./gradlew installDebug`). Once you reach your React-powered activity inside the app, it should load the JavaScript code from the development server and display:
 
 ![Screenshot](img/EmbeddedAppAndroid.png)
-
-## Sharing a ReactInstance across multiple Activities / Fragments in your app
```

Comment:
Yes please revert this. It's not super up-to-date but I think it's worth leaving in as a starting point.